### PR TITLE
[Windows] Add VC components for VS 17.7

### PR DESCRIPTION
**Pull request by vpolikarpov-akvelon**
# Description
Add missing build tools for Visual Studio 2022 v17.7

## Check list
- [ ] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [x] Changes are tested and related VM images are successfully generated

### CODE REVIEW

**Comment by Tractorou24** [comment]
+1 on this PR, without this version we can't compile MSVC STL std::source_location with clang!

File: images/win/toolsets/toolset-2022.json [hunk 1]
```diff
@@ -283,6 +283,24 @@
             "Microsoft.VisualStudio.Component.VC.14.36.17.6.MFC",
             "Microsoft.VisualStudio.Component.VC.14.36.17.6.x86.x64.Spectre",
```
[the remark this answers is not in the document]
Comment (vpolikarpov-akvelon) [reply images/win/toolsets/toolset-2022.json]: Applied, thanks

**Review by mikhailkoliada — APPROVED** [review]

**Pull request merged**